tests/stub/authorization/scripts/v4x0/reader_return_1_failure_return_2_and_3_succeed.script

```diff
@@ -18,8 +18,10 @@ A: HELLO {"{}": "*"}
             S: RECORD [1]
                SUCCESS {"type": "r", "has_more": true}
         +}
-        C: DISCARD {"n": -1}
-        S: SUCCESS {}
+        {?
+            C: DISCARD {"n": -1}
+            S: SUCCESS {}
+        ?}
```

Comment:
Now that I think about it, shouldn't it be
```suggestion
        {{
            C: DISCARD {"n": -1}
            S: SUCCESS {}
            ----
            A: RESET
        }}
```

If I recall this correctly, it is the JS driver sending a `RESET` instead of a `DISCARD`
https://trello.com/c/xHx42X4N/43-discard-or-reset-on-unfinished-result
And one *could* argue, that RESET is actually the wrong thing here, as it aborts the query. Not 100% sure.

Reply:
Actually, the issue only warns in the log, the test was not failing because JS doesn't care if the reset fails, it will not thrown the error, it will just drop the connection from the pool.